### PR TITLE
feat(installation): define when/how to fire BIP

### PR DESCRIPTION
**Pull request by marcoscaceres**

### CODE REVIEW

**Comment by marcoscaceres** [comment]
Ok, once this final one lands, then I'm going to do some editorial reorganization of the editorial section. But let's get this in and let's get implementing!!! ❤️

File: index.html [hunk 1]
```diff
@@ -374,6 +360,67 @@
         </ol>
       </section>
       <section>
+        <h2>
+          Install prompts
+        </h2>
+        <p>
+          An end-user can <dfn data-lt="manual installation">manually</dfn>
```
Comment (kenchris) [inline index.html]: is that a requirement that that is possible?

**Comment by marcoscaceres** [comment]
WARNING: IF THIS DOESN'T LAND BY TOMORROW TRUMP GETS AUTOMATICALLY ELECTED!

**Comment by kenchris** [comment]
Does that means that if he gets elected, you drop this patch?

**Comment by marcoscaceres** [comment]
> Does that means that if he gets elected, you drop this patch?

Will it matter at that point?

**Comment by kenchris** [comment]
im fine with this (patch that is)

File: index.html (hunk 1 above)
Comment (marcoscaceres) [inline index.html]: No, it's just a statement of fact.

**Comment by marcoscaceres** [comment]
@mgiuca see https://github.com/w3c/manifest/pull/520

**Pull request merged**